docs/cloud-dev-env.md

```diff
@@ -0,0 +1,26 @@
+# Cloud Development Environments
+
+Altis has experimental built-in support for [GitHub Codespaces](https://github.com/features/codespaces), a Cloud-based development environment, that makes it possible to spin up a complete development environment in seconds within your browser, with a full editor based on Visual Studio Code and a fully-featured environment for your project, without having to install anything on your local machine.
+
+This is useful for many reasons:
+    - Few seconds to bootstrap a full development environment.
+    - Zero local installation required.
+    - Standardized and highly configurable environment setup.
+    - Powerful development environment, scalable as needed.
```

Comment:
What exactly is this saying? Can you scale the available memory / cpu?

i would drop the “as needed”

Reply:
You can choose the specs of the VM, yes. The `as needed` bit is to express there is a standard setup, depending on our logistics still.

Reply:
```suggestion
    - Powerful and scalable development environment.
```